### PR TITLE
Gardening: Migrate compiler sources to GitHub issues

### PR DESCRIPTION
**Pull request by AnthonyLatsis**
I am intentionally violating the 80 char limit in a lot of places here. The idea is for `TODO` and `FIXME` to show up nicely in search results, context and all, no truncation. Doesn’t look too bad with line wrapping enabled. Also, perhaps unfortunately, Xcode doesn’t pick up on `TODO(...):` or `FIXME(...):`. Thoughts?

### CODE REVIEW

File: lib/ClangImporter/ImportDecl.cpp [hunk 1]
```diff
@@ -3006,8 +3006,7 @@ namespace {
 
       auto templateParamTypeUsedInSignature =
           [decl](clang::TemplateTypeParmDecl *type) -> bool {
-        // TODO(SR-13809): we will want to update this to handle dependent
-        // types when those are supported.
+        // TODO: we will want to update this to handle dependent types when those are supported. (https://github.com/apple/swift/issues/56206)
```
Comment (xedin) [inline lib/ClangImporter/ImportDecl.cpp:3009]: I'd vote for `TODO`, `FIXME`, and `MARK` mentioning issue in parentheses if that's not too much work to make it happen...

Comment (AnthonyLatsis) [reply lib/ClangImporter/ImportDecl.cpp:3009]: I find it less discoverable, but I don’t mind. What about the char limit?

Comment (xedin) [reply lib/ClangImporter/ImportDecl.cpp:3009]: I think it’s okay to go over the 80 char limit in such circumstances.

**Review by xedin — APPROVED** [review]

**Comment by AnthonyLatsis** [comment]
@swift-ci please test

**Comment by AnthonyLatsis** [comment]
@swift-ci please test Windows

**Pull request merged**